kafka/pykafka/topic.py

```diff
@@ -36,11 +35,18 @@ def name(self):
     def partitions(self):
         return self._partitions
 
-    def earliest_offsets(self):
-        """Get the earliest offset for each partition of this topic."""
-        return self.fetch_offsets(OFFSET_EARLIEST)
+    def get_producer(self):
+        return Producer(self)
+
+    def fetch_offset_limits(self, offsets_before, max_offsets=1):
+        """Use the Offset API to find a limit of valid offsets
+            for each partition in this topic
```

Comment:
Total nitpick, but the standard is to have the first line be a short, single-line description of the function.  The hanging indent is just confusing to me. :)